android/src/main/java/com/taplytics/react/TaplyticsReactModule.java

```diff
@@ -283,6 +289,9 @@ public void sessionInfoRetrieved(HashMap hashMap) {
 
                 callback.resolve(resultData);
             }
+            public void onError(HashMap hashMap) {
+                callback.resolve(null);
```

Comment:
should we reject the promise?

Reply:
@jonathannorris not sure what rejecting the promise does inside of the react package.  Need to ask @ajwootto about that. 

@Arpit-Patel This should be resolved with an empty hashmap I believe

Reply:
well rejecting the promise is basically the same as an error response.

Reply:
yeah I think this should probably reject with an error message